cpt/ci_manager.py

```diff
@@ -216,10 +216,12 @@ def get_commit_id(self):
         return os.getenv("CIRCLE_SHA1", None)
 
     def get_branch(self):
+        if self.is_pull_request():
+            return None
         return os.getenv("CIRCLE_BRANCH", None)
 
     def is_pull_request(self):
-        return os.getenv("CIRCLE_PULL_REQUEST", None)
+        return os.getenv("CIRCLE_PULL_REQUEST", None) is not None
```

Comment:
Maybe `return os.getenv("CIRCLE_PULL_REQUEST", False)`

Reply:
Actually, `os.getenv("CIRCLE_PULL_REQUEST", False)` does not return True when the environment variable is set, so it makes the test fail (`self.assertEquals(manager.is_pull_request(), False)`). alternatively we could replace the test with ` self.assertIsNotNone`, but it feels more natural that a function named is_*** returns a boolean value

Reply:
you are right.